tests/index.test.ts

```diff
@@ -1,11 +1,63 @@
-import { Document } from "gherkin-ast";
-import { format, FormatOptions } from "../src";
+import * as fs from "fs";
+import * as path from "path";
+
+import { Document, Feature } from "gherkin-ast";
+import { read } from "gherkin-io";
+import { format } from "../src";
+
+const readFile = (fileName: string) => fs.readFileSync(path.resolve(`tests/testData/${fileName}.feature`), "utf8");
+const parse = async (fileName: string) => await read(`tests/testData/${fileName}.feature`);
 
 describe("gherkin-formatter", () => {
-    test("should pass", () => {
-        const document: Document = new Document("uri");
-        const options: FormatOptions = { lineBreak: "\n" } as FormatOptions;
-        expect(format(document, options))
-            .toBe("{\"document\":{\"uri\":\"uri\",\"feature\":null},\"options\":{\"lineBreak\":\"\\n\"}}");
-    });
+  // tslint:disable-next-line: no-any
+  const toFormat: any = {};
+  // tslint:disable-next-line: no-any
+  const expected: any = {};
+
+  beforeAll(async () => {
+    toFormat.base = await parse("base");
+    expected.base = readFile("base");
+    toFormat.background = await parse("background");
+    expected.background = readFile("background");
+    toFormat.emptyFeature = await parse("emptyFeature_toParse");
+    expected.emptyFeature = readFile("emptyFeature_toParse");
+    toFormat.emptyRule = await parse("emptyRule_toParse");
+    expected.emptyRule = readFile("emptyRule_toParse");
+    toFormat.oneTagPerLine = await parse("newLineOption_toParse");
+    expected.oneTagPerLine = readFile("newLineOption_expected");
+    toFormat.separateStepGroups = await parse("separateStepGroups_toParse");
+    expected.separateStepGroups = readFile("separateStepGroups_expected");
+    toFormat.compact = await parse("compact_toParse");
+    expected.compact = readFile("compact_expected");
+  });
+
+  it("should format simple document", async () => {
+    expect(format(toFormat.base[0]).split(/\r?\n/g)).toEqual(expected.base.split(/\r?\n/g));
+  });
+  it("should throw error if the given object is not a GherkinDocument", async () => {
+    const feature = new Feature("S1", "S2", "S3");
+    expect(() => format({uri: "string", feature} as Document))
```

Comment:
Please apply formatting to the test-files as well, this formatting does not seem to be correct.